src/tests/sqlsmith/src/sql_gen/types.rs

```diff
@@ -33,6 +33,7 @@ pub(super) fn data_type_to_ast_data_type(data_type: &DataType) -> AstDataType {
         DataType::Int16 => AstDataType::SmallInt,
         DataType::Int32 => AstDataType::Int,
         DataType::Int64 => AstDataType::BigInt,
+        DataType::Serial => AstDataType::Custom(vec!["SERIAL".into()].into()),
```

Comment:
This should be `unreachable`. `sqlsmith` should not generate a **user query** containing `SERIAL` data type word, which we do not have corresponding parsing logic in binder intentionally.